### PR TITLE
FOGL-1664 Cleanup init.sql and remove predefined entries for services…

### PR DESCRIPTION
**Pull request by MarkRiddoch**
… and tasks

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,22 +14,24 @@ add_subdirectory(C/plugins/storage/postgres)
...
+		add_subdirectory(C/plugins/south/modbus)
+	endif()
+	add_subdirectory(C/plugins/north/omf)
+	add_subdirectory(C/plugins/north/ThingSpeak)
+	add_subdirectory(C/plugins/north/http-north)
+	add_subdirectory(C/plugins/filter/scale)
+endif (BUILD_C_PLUGINS)
 add_subdirectory(C/plugins/utils)
-add_subdirectory(C/plugins/filter/scale)
 # Add Python 2.7 filter
```
Comment (pintomax) [inline CMakeLists.txt]: C/plugins/filter/python27 is a C++ filter which embeds Python 2.7 intepreter

It should be moved into into the BUILD_C_PLUGINS conditional build as well

Comment (MarkRiddoch) [reply CMakeLists.txt]: This doesn't matter as filter plugins are not seen by the Python code.

Comment (pintomax) [reply CMakeLists.txt]: We ca move out add_subdirectory(C/plugins/filter/scale) then

File: scripts/plugins/storage/sqlite/downgrade/8.sql [hunk 2]
```diff
@@ -0,0 +1,145 @@
+
+rth plugins
```
Comment (praveen-garg) [inline scripts/plugins/storage/sqlite/downgrade/8.sql]: @MarkRiddoch  @pintomax  this may result in error?

**Pull request merged**